src/main/java/apaintus/services/draw/selection_box/SelectionBoxDrawService.java

```diff
@@ -0,0 +1,19 @@
+package apaintus.services.draw.selection_box;
```

Comment:
why should the drag_select have its own service when we have the drawService ?

Reply:
Parce que il y a une implementation unique a chaque truc qu'on dessine. Voir strategy pattern

Reply:
then on a bleeder un peu pas mal partout alors dans le code si on suis cette logique

Reply:
Bleeder où?
Et j'en avais besoin d'un qui ne faisait rien puisque je ne voulais que print la bounding box.

Reply:
je ne comprend pas ton commentaire

Reply:
en ce moment drawService est appeler pour draw des shapes, le BoundingBox(qui est un shape, mais bon) et le snapgrid. 3 choses differentes l'un de lautre. Le SelectionBoxDrawService servirais a adder un Bounding box par desuss la selection du dragSelect ce qui m'apparait comme etant un service qu'offre deja le drawService pour les bounding box, est-ce que je me trompe?

Reply:
J'ai voulu utiliser que le drawService du boundingBox pour la selectionBox, mais le canvasService appelle le drawService de la bounding box de la selected shape. Donc j'aurais introduit une redondance d'appel inutile à cet endroit, donc j'ai défini un drawService pour la Selection Box qui s'assure que rien n'est appelé sauf le drawService de la bounding box par le canvasService.

Reply:
ok

Reply:
@pspino DrawService c'est une classe abstraite tu peux pas l'instantier pis l'utiliser